sdk/rust/src/account/account_create_transaction.rs

```diff
@@ -80,6 +83,10 @@ pub struct AccountCreateTransactionData {
     ///
     pub max_automatic_token_associations: u16,
 
+    pub alias_key: Option<PublicKey>,
+
+    pub alias_evm_address: Option<[u8; 20]>,
```

Comment:
`alias_evm_address` is now just `evm_address` right?

Reply:
Yep.  Good catch